### PR TITLE
Implemented patient appointment recommendation

### PR DESCRIPTION
**Pull request by nikolapantelic-ftn**
- Added component for appointment recommendation on front-end
- Implemented services and API controllers
- Added unit and integration tests

### CODE REVIEW

File: HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        formAppointmentString:function(appointment){
+            
+            //TODO: format dropdown list item
+            /*
+            let appointmentStart=moment(timeInterval.start)
+            let appointmentEnd=moment(timeInterval.end)
+            
+            return appointmentStart.format("dddd, MMMM Do YYYY") +
+             " "+appointmentStart.format("h:mm")+" - "+appointmentEnd.format("h:mm")
+            */
```
Comment (rammba) [inline HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue:63]: Is this comment needed for todo feature or?

Comment (sStrbacki) [reply HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue]: Removed in latest commit.

**Comment by rammba** [comment]
[(1) PR Implemented patient appointment recommendation](https://trello.com/c/Jb6HnXVD/114-1-pr-implemented-patient-appointment-recommendation)

File: HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue [hunk 2]
```diff
@@ -53,15 +57,22 @@ export default {
...
-            let appointmentEnd=moment(timeInterval.end)
             
-            return appointmentStart.format("dddd, MMMM Do YYYY") +
-             " "+appointmentStart.format("h:mm")+" - "+appointmentEnd.format("h:mm")
-            */
+            let appointmentStart=moment(appointment.timeInterval.start)
+            let appointmentEnd=moment(appointment.timeInterval.end)
+            
+            return "Doktor: " + appointment.doctor.person.name + " " + appointment.doctor.person.surname + " - "
+             +appointmentStart.format("dddd, MMMM Do YYYY") +
```
Comment (rammba) [inline HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue:65]: Is this string on english or on serbian?

Comment (nikolapantelic-ftn) [reply HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue]: Changed to english on all occurrences.

**Review by BoKna031 — APPROVED** [review]

**Review by rammba — APPROVED** [review]

**Comment by nikolapantelic-ftn** [comment]
[[E2.2] As a patient I want to schedule an appointment using a recommendation so that I could find an appointment easier. [5]](https://trello.com/c/TsVbse6Y/31-e22-as-a-patient-i-want-to-schedule-an-appointment-using-a-recommendation-so-that-i-could-find-an-appointment-easier-5)

**Pull request merged**